requirements-dev.txt

```diff
@@ -1,4 +1,4 @@
-colorama==0.4.6 ; sys_platform == "win32" and python_full_version == "3.8.10"
+colorama==0.4.6 ; python_full_version == "3.8.10" and sys_platform == "win32"
```

Comment:
Can we dig into this?  This seems really weird.

Reply:
```
% poetry show colorama
 name         : colorama                              
 version      : 0.4.6                                 
 description  : Cross-platform colored terminal text. 

required by
 - pytest *
```

So it is there so pytest can display things in color (even on win)

Reply:
got it so colorama is for pytest incase it's installed on windows.  Annoying.